ibm/mas_devops/roles/db2_backup/README.md

```diff
@@ -33,6 +39,8 @@ Example Playbook
 - hosts: localhost
   any_errors_fatal: true
   vars:
+    db2_namespace: "db2u"
```

Comment:
@cuddlyporcupine  To be consistent with default values used in db2 role, I would suggest we default namespace in backup/restore roles with same value as [here](https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/db2/defaults/main.yml#L4)